### PR TITLE
Update scripts in "Build the project"

### PR DESCRIPTION
**Pull request by xhlulu**
This was mentioned in the following issues:
* Closes https://github.com/plotly/dash-component-boilerplate/issues/96
* Closes https://github.com/plotly/dash-docs/issues/1036

cc: @ndrezn 

The old scripts:

> - `npm run build:js-dev` generate the development bundle `project_shortname.dev.js`, use with `app.run_server(debug=True)`
> - `npm run build:py` generate the python classes files for the components.
> - `npm run build:all` generate both bundles and the languages classes files.

The new scripts:

> - `npm run build:py_and_r` generate the python classes files for the components.
> - `npm run build` generate both bundles and the languages classes files.

This matches the scripts defined in [package.json](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/package.json)

---

Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [x] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

If this PR adds an image or animated GIF:
- [ ] This image was saved and referenced locally rather than via an external link

If I introduced a new relative link inside `dcc.Markdown`:
- [ ] I considered whether I could replace the `dcc.Markdown` call with `rc.Markdown`, which will replace relative links with `tools.relpath` internally. Otherwise, I used e.g. `<dccLink href=tools.relpath('/layout') children="the first chapter"/>` instead of `[the first chapter](/layout)` (importing `tools` from `dash_docs`), and set `dangerously_allow_html=true` in the `dcc.Markdown` call.

If I changed the `chapter_index` by removing or relocating a page:
- [ ] I added a redirect in `dash_docs/server.py` from the old URL to the new URL

### CODE REVIEW

File: dash_docs/chapters/react_for_python_developers/index.py [hunk 1]
```diff
@@ -143,9 +143,8 @@
 ### Build the project
 
 - `npm run build:js` generate the production bundle `project_shortname.min.js`
```
Comment (alexcjohnson) [inline dash_docs/chapters/react_for_python_developers/index.py:145]:
```suggestion
- `npm run build:js` generate the JavaScript bundle `project_shortname.min.js`
```
Now that there aren't separate prod/dev bundles - if you want we could mention the `.js.map` file but that's probably not important enough for this part.

File: dash_docs/chapters/react_for_python_developers/index.py [hunk 2]
```diff
@@ -143,9 +143,8 @@
...
-- `npm run build:js-dev` generate the development bundle `project_shortname.dev.js`, use with `app.run_server(debug=True)`
-- `npm run build:py` generate the python classes files for the components.
-- `npm run build:all` generate both bundles and the languages classes files.
+- `npm run build:py_and_r` generate the python classes files for the components.
+- `npm run build` generate both bundles and the languages classes files.
```
Comment (alexcjohnson) [inline dash_docs/chapters/react_for_python_developers/index.py:147]:
```suggestion
- `npm run build:py_and_r` generate the Python and R class files for the components.
- `npm run build` generate everything: the JavaScript bundles and the Python and R class files.
```

**Comment by xhlulu** [comment]
Thanks @alexcjohnson I committed your suggestions

**Review by alexcjohnson — APPROVED** [review]
💃 Thanks for cleaning this up!

**Pull request merged**